Solinea/goldstone-server
# password reset via /accounts/password/reset/ not working in containerized environment

This fixes the bug that "reset password" e-mail isn't being sent.

This does *not* fix the problem of the link in the "reset password" e-mail not working in our containerized environment, because that's in a separate ticket.

To test this:

1. If you don't have `ghmarkdown` installed on your laptop, install it. You must install version 1.0.5, because that's the last version that supports Python 2.*:

        $ pip install ghmarkdown==1.0.5

2. Switch to the reset branch and create the HACKING html document:

        $ workon goldstone-server
        $ git checkout reset
        $ ghmarkdown --i docs/HACKING.md --o foo.html

3. Open `foo.html` in your favorite browser.

4. Search for the section titled, "Configuring Postfix."  Follow the instructions.

You should be able to send yourself a "password reset" e-mail.
